pom.xml

```diff
@@ -829,114 +829,52 @@
     </build>
     <profiles>
         <profile>
-            <id>advanced</id>
```

Comment:
We cannot just delete the advanced profile. We need to move the executions of maven assembly plugin

Reply:
There should be also a way to suppress creation of .zip file (just generate .jar).